### PR TITLE
Add handling of verification form submit

### PR DESCRIPTION
**Pull request by vzajkov**
## Description
_Remove this and add a detailed description of the changes that were made._

## Test Plan
_Remove this and add a test plan that the reviewer can use to test your changes._

### CODE REVIEW

**Comment by vzajkov** [comment]
@karenmrose  - passing in values via the `updateResource` call on submit, however getting a 500 error on verification form submit. Is the endpoint expecting different data here?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
- advances
+  advances
```
Comment (karenmrose) [inline CODE_OF_CONDUCT.md]: @vzajkov looks like this file got added. You might need to just revert this change.

File: package.json [hunk 2]
```diff
@@ -9,13 +9,16 @@
 	},
 	"scripts": {
 		"build:html": "babel-node client/tools/build-html.js",
-		"clear-iv": "NODE_ENV=localhost node ./server/scripts/clear-in-verification-field.js",
+		"clear-iv":
```
Comment (karenmrose) [inline package.json]: @vzajkov Looks like this file got added too in the merge.

**Comment by karenmrose** [comment]
Fixed via this PR: https://github.com/HelpAssistHer/help-assist-her/pull/188
